app/src/main/res/values/strings.xml

```diff
@@ -383,6 +383,18 @@
         <item>+/- 0.5</item>
     </string-array>
 
+    <string-array name="wave_generator">
+        <item>W1</item>
+        <item>W2</item>
+    </string-array>
+
+    <string-array name="voltage_source">
+        <item>PV1</item>
+        <item>PV2</item>
+        <item>PV3</item>
+        <item>PCS</item>
```

Comment:
Yes. I know it is a current source. I wanted to use them in the same spinner. I just kept the name as `voltage_source`. I can change the name. Suggest an alternative?